packages/react/src/autocomplete/Autocomplete.test.tsx

```diff
@@ -62,9 +62,10 @@ describe('Autocomplete', () => {
 		expect(input).toHaveAttribute('aria-expanded', 'false');
 		if (!input) return;
 
-		// Focus the input
```

Comment:
In ARIA version 2, focusing the input should not open the combobox anymore. Therefor, these tests need to be updated slightly.

Reply:
Suggest adding another `expect(input).toHaveAttribute('aria-expanded', 'true')` once it is clicked?